SConstruct

```diff
@@ -746,7 +751,7 @@ system_dependencies = all_dependencies - download_dependencies
 if 'libuv' in system_dependencies:
     conf = Configure(env, custom_tests=env.CustomTests)
 
-    env.ParsePkgConfig('--cflags --libs libuv')
+    conf.AddPkgConfigDependency('libuv', '--cflags --libs')
```

Comment:
We also should use AddPkgConfigDependency for google-benchmark.

Reply:
Also, can we remove the old `env.ParsePkgConfig` method?

https://github.com/roc-project/roc/blob/05b998cce82fcc4e6ba9de96ab68821075747606/site_scons/site_tools/roc/parsers.py#L99-L111